pom.xml

```diff
@@ -19,13 +18,19 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
you used a different library here - does your logging work?
please share a screenshot with log output to console and file when you run your app